database/insertData/_common/03_template_C_userRegistration.js

```diff
@@ -150,11 +418,6 @@ exports.queries = [
             create: [
               # No Core Actions for this one
               # ${devActions}
```

Comment:
Can probably remove "devActions" for "production" templates -- we can do everything we need with the `dev` templates.

Reply:
Thanks. Removed: 23639b5